clustered/client/src/main/java/org/ehcache/clustered/client/internal/service/ValueCodecFactory.java

```diff
@@ -39,7 +41,7 @@
         || clazz.isPrimitive() || String.class.equals(clazz)) {
```

Comment:
Should we care if a primitive wrapper class hasn't been whitelisted?

Reply:
This is more of an optimisation where we do not serialise JDK classes as we can resolve them on the server and their immutability makes them safe.